vite.config.ts

```diff
@@ -89,6 +90,7 @@ export default defineConfig(({ mode }) => {
 
   if (buildLibrary) {
     // Config for deployment in SciGateway
+    plugins.push(cssInjectedByJsPlugin());
```

Comment:
Just a note: If we go to ESM this will need to change. I created an issue to keep track https://github.com/ral-facilities/inventory-management-system/issues/1086.